js/server/modules/@arangodb/statistics.js

```diff
@@ -571,29 +571,29 @@ exports.installPeriodicTasks = function () {
   var interval = exports.STATISTICS_INTERVAL;
   var interval15 = exports.STATISTICS_HISTORY_INTERVAL;
 
-  internal.registerTask({
-    id: 'statistics-collector',
-    name: 'statistics-collector',
-    offset: interval / 10,
-    period: interval,
-    command: "require('@arangodb/statistics').historian();"
-  });
-
-  internal.registerTask({
-    id: 'statistics-average-collector',
-    name: 'statistics-average-collector',
-    offset: 2 * interval,
-    period: interval15,
-    command: "require('@arangodb/statistics').historianAverage();"
-  });
-
-  internal.registerTask({
-    id: 'statistics-gc',
-    name: 'statistics-gc',
-    offset: Math.random() * interval15 / 2,
-    period: interval15 / 2,
-    command: "require('@arangodb/statistics').garbageCollector();"
-  });
+  // internal.registerTask({
```

Comment:
Please remove these comments before Merge

Reply:
what happens to the .js file anyway?